docs/cadence/language/attachments.mdx

```diff
@@ -111,6 +120,31 @@ access(all) attachment SomeAttachment for SomeContract.SomeStruct {
 }
 ```
 
+By default, the `base` reference is unauthorized, and thus entitled-access members on the base type are inaccessible in the attachment. 
+If the author of the attachment wishes to have access to entitled-access members on the base type, 
+they must declare that their attachment requires certain entitlements to the base, using `require entitlement E` syntax.
+Required entitlements must be valid entitlements for the base type,
+and requiring entitlements in the attachment declaration will impose additional requirements when the attachment is attached, 
+as described below. So, for example:
+
+```cadence
+entitlement mapping M {
+    E -> F
+}
+resource R {
+    access(E) fun foo() { 
+        //...
+    }
+}
+access(M) attachment A for R {
+    require entitlement E
+
+    access(all) fun bar() {
+        base.foo() // available because `E` is required above, and thus `base` is type `auth(E) &R`. 
+    }
+}
```

Comment:
```suggestion
entitlement mapping M {
    E -> F
}

resource R {
    access(E) fun foo() { 
        //...
    }
}

access(M) attachment A for R {
    require entitlement E

    access(all) fun bar() {
        base.foo() // available because `E` is required above, and thus `base` is type `auth(E) &R`. 
    }
}
```